.github/workflows/cloud_chatops.yaml

```diff
@@ -0,0 +1,63 @@
+name: CI/CD Cloud ChatOps
+
+on:
+  push:
+    branches:
+      - "master"
+
+jobs:
+  Pylint-Tests-Codecov:
+    runs-on: ubuntu-20.04
+    strategy:
+      matrix:
+        python-version: [ "3.x" ]
```

Comment:
Did you have a version of Python which you originally built against? 
We should test a known good, and the latest version in the matrix so it's obvious if a failure is from bleeding edge or the proposed change

Reply:
Good catch, I did not think about that.